src/features/defi/providers/fox-farming/components/FoxFarmingManager/Deposit/components/Deposit.tsx

```diff
@@ -257,16 +254,6 @@ export const Deposit: React.FC<DepositProps> = ({
     [asset, cryptoBalance, marketData.price],
   )
 
-  const handleBack = useCallback(() => {
-    history.push({
-      pathname: `/defi/earn`,
-      search: qs.stringify({
-        ...query,
-        modal: DefiAction.Overview,
-      }),
-    })
-  }, [history, query])
-
```

Comment:
q: what will be the back behaviour now that this is rm'd?

Reply:
We don't have any back behavior on these modals, handleBack was not consumed it was dead code